docs/man/git-lfs-migrate.1.ronn

```diff
@@ -84,13 +84,39 @@ options and these additional ones:
     Write to 'path' a file with the mapping of each rewritten commits. The file
     format is CSV with this pattern: `OLD-SHA`,`NEW-SHA`
 
-If `--include` or `--exclude` (`-I`, `-X`, respectively) are given, the
+* `--no-rewrite`
+    Migrate large objects to Git LFS in a new commit without rewriting git history.
+    Please note that when this option is used, the `migrate import` command will expect
+    a different argument list, specialized options will become available, and the core 
+    `migrate` options will be ignored. See [IMPORT (NO REWRITE)].
+
+If `--no-rewrite` is not provided and `--include` or `--exclude` (`-I`, `-X`, respectively) are given, the
 .gitattributes will be modified to include any new filepath patterns as given by
 those flags.
 
-If neither of those flags are given, the gitattributes will be incrementally
+If `--no-rewrite` is not provided and neither of those flags are given, the gitattributes will be incrementally
 modified to include new filepath extensions as they are rewritten in history.
 
+### IMPORT (NO REWRITE)
+
+The `import` mode has a special sub-mode enabled by the `--no-rewrite` flag. This sub-mode
+will migrate large objects to pointers as in the base `import` mode, but will do so in a new commit
+without rewriting Git history. When using this sub-mode, the base `migrate` options, 
+such as `--include-ref`, will be ignored, as will those for the base `import` mode. 
+The `migrate` command will also take a different argument list.
+
+The `--no-rewrite` sub-mode supports the following options and arguments:
+
+* `-m` <message> `--message`=<message>
```

Comment:
I think the backticks should wrap from the first `-` to the last `>` in each example.